test/java/seedu/addressbook/logic/CommandAssertions.java

```diff
@@ -100,7 +127,7 @@ public static void assertCommandBehavior(String inputCommand,
     }
 
     /**
-     * Executes the command and confirms that the result message is correct and
+     * Executes the command and confirms that the result messages are correct and
      * Assumes the command does not write to file
      * MessageType specify which console the message is supposed to be written to
      * @see #assertCommandBehavior(String, String, AddressBook, boolean, List, boolean)
```

Comment:
Also the same as above... Hmmm maybe it'll be wise to specify those the allows you to choose which console to output on

Reply:
added a line for the MessageType param